src/templates/token.tsx

```diff
@@ -97,32 +120,124 @@ const generateTokenPage = ({ secret }: { secret: string }) => {
             </MjmlText>
           </MjmlColumn>
         </MjmlSection>
-        <MjmlSection mjClass="bg-secondary" padding="1px" borderRadius="16px">
+        <MjmlSection
+          mjClass="bg-secondary"
+          padding={values.px}
+          borderRadius={values.px16}
+        >
           <MjmlColumn
-            paddingLeft="24px"
             mjClass="bg-primary-dark"
-            borderRadius="16px"
+            padding={values.px16}
+            borderRadius={values.px16}
           >
             <MjmlText
               mjClass="font-accent text-formal-accent"
-              paddingTop="32px"
-              fontSize="20px"
+              paddingTop={values.px32}
+              fontSize={values.px20}
               fontWeight={700}
             >
               Your token is:
             </MjmlText>
             <MjmlText
               mjClass="font-accent text-secondary"
-              paddingBottom="32px"
-              paddingTop="16px"
-              fontSize="16px"
+              paddingBottom={values.px32}
+              paddingTop={values.px16}
+              fontSize={values.px16}
               fontWeight={700}
             >
               {secret}
             </MjmlText>
           </MjmlColumn>
         </MjmlSection>
-        <MjmlSection paddingBottom="32px"></MjmlSection>
+
+        <MjmlSpacer height={values.px64} />
+
+        {/* Body */}
+        <MjmlSection
+          padding={values.px32}
+          borderRadius={values.px16}
+          backgroundColor={colors.primary}
+        >
+          <MjmlColumn>
+            <MjmlImage
+              width={values.px64}
+              src={`${assetsEndpoint}/wow_seal.png`}
+            />
+            <MjmlSpacer height={values.px16} />
+            <MjmlText
+              mjClass="font-bold text-secondary"
+              fontSize={values.px24}
+              color={colors.formal}
+              fontWeight={700}
+              align="center"
+            >
+              What now?
+            </MjmlText>
+            <MjmlSpacer height={values.px8} />
+            <MjmlText
+              mjClass="font-accent text-secondary"
+              fontSize={values.px14}
+              line-height={values.px22}
+              fontWeight={400}
+              align="center"
+              color={colors.formal}
+            >
+              Copy the token above and bring it back to SealCred. From there,
+              you can create your ZK proof and mint a ZK badge that you can add
+              to your anonymous wallet.
+            </MjmlText>
+            <MjmlSpacer height={values.px40} />
+            <MjmlImage src={`${assetsEndpoint}/badge_flow.png`} width="377px" />
```

Comment:
Here we use image for `mobile` flow, because it looks fine on mobiles and desktops, as [you can see](https://www.caniemail.com/search/?s=display), the `display: none` doesn't work properly for images, I tried to wrap it in `<div>`, but that didn't work too
![image](https://user-images.githubusercontent.com/49658988/177817188-8847db05-0bb6-4071-9592-a03550bf9f34.png)